deploy/roles/r-libs/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install packages required by R packages
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ item }}"
+    state: present
+    disable_gpg_check: yes
+  with_items: "{{ r_dependencies }}"
+  tags:
+    - r-update
```

Comment:
@lel99999 The `tag:` normally should just match the role name to keep it simple, so `r-libs` instead of `r-update`.